content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+
+**Note:** Configuring values for all the components of KKP is a humongous task. Also - each customer might like a different directory structure to manage KKP installation. This ArgoCD Apps based approach is an _opinionated attempt_ to provide a standard structure that can be used in most of the customer installations. If you need different directory structure, refer to [README.md of ArgoCD Apps Component](https://github.com/kubermatic/kubermatic/blob/main/charts/gitops/kkp-argocd-apps/README.md) to understand how you can customize this, if needed.
+
+### ArgoCD Apps
+We will install ArgoCD on both the clusters and we will install following components on both clusters via ArgoCD. In non-GitOps scenario, some of these components are managed via kubermatic-installer and rest are left to be managed by KKP administrator in master/seed clusters by themselves. With ArgoCD, except for kubermatic-operator, everything else can be managed via ArgoCD. Choice remains with KKP Administrator to include which apps to be managed by ArgoCD.
+
+1. Core KKP components
+    1. DEX (in master)
+    1. ngix-ingress-controller
+    1. certManager
+1. Backup components
+    1. Velero
+1. Seed monitoring tools
+    1. Prometheus
+    1. AlertManager
+    1. Grafana
+    1. KubeStateMetrics
+    1. NodeExporter
+    1. BlackboxExporter
+    1. Identity aware proxy (IAP) for seed monitoring  components
+1. Logging components
+    1. Promtail
+    1. Loki
+1. S3 like object storage - minio
+1. User-cluster mla components
+    1. Minio and Minio Lifecycle Manager
+    1. Grafana
+    1. Consul
+    1. Cortex
+    1. LokiDistributed
+    1. AlertmanagerProxy
+    1. iap for user-mla
+    1. secrets - grafana and minio secrets
+1. Seed Settings - Kubermatic configuration, Seed objects, preset objects and such misc objects needed for Seed configuration
+1. Seed Extras - This is a generic ArgoCD app to deploy arbitrary resources not covered by above things and as per needs of KKP Admin.
+
+## Installation
+
+> You can find same tutorial code with sample values in [this git repository](https://github.com/dharapvj/kkp-using-argocd). For ease of installation, a `Makefile` has been provided to just make commands easier to read. Internally, it just depends on helm, kubectl and kubermatic-installer binaries. But you will need to look at `make` target definitions in `Makefile` to adjust DNS names. While for the demo, provided files would work, you would need to look through each file under `dev` folder and customize the values as per your need.
+
+### Setup two Kubernetes Clusters
+> This step install two Kubernetes clusters using Kubeone in AWS. You can skip this step, if you already have access to two kubernetes clusters.
```

Comment:
* KubeOne with a capital O
* "You can skip this step, if you" without the comma
* Kubernetes capitalized